demo/src/main/res/layout/activity_accessibility_checker.xml

```diff
@@ -1,17 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  android:orientation="vertical">
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
open closed principle innit - if we need to add another 10-20 services, just need to add a textview each time :D